### ISSUE TITLE
Other language

### ISSUE THREAD

**Issue by skyepodium** [Kyubyong/tacotron#94]
Is it possible to use tactron for other languages(ex: korean)

**Comment by Jackiexiao** [comment]
Is there any one try to use tacotron for other language?

**Comment by Kyubyong** [comment]
Absolutely. I'm doing that for 10 non-English languages. So far so good.

**Comment by japita-se** [comment]
how to use other languages?

**Comment by zyb8543d** [comment]
anyone know whether the tool is used for Chinese?

**Comment by makerjackie** [comment]
so far as I know, in other implementations of tacotron, someone have tested it for mandarin tts, see https://github.com/keithito/tacotron/issues/118, this project can used for other language too though I haven't test it

**Comment by zyb8543d** [comment]
@Jackiexiao thanks, i will try it on chs too

**Comment by ghost** [comment]
> "Absolutely. I'm doing that for 10 non-English languages. So far so good."

what are the changes required for other languages?
